easybuild/easyconfigs/h/Hypre/Hypre-2.10.1-intel-2015b.eb

```diff
@@ -0,0 +1,22 @@
+name = 'Hypre'
+version = '2.10.1'
+
+homepage = "https://computation.llnl.gov/casc/linear_solvers/sls_hypre.html"
+description = """Hypre is a library for solving large, sparse linear systems of equations on massively parallel computers.
+ The problems of interest arise in the simulation codes being developed at LLNL and elsewhere 
+ to study physical phenomena in the defense, environmental, energy, and biological sciences."""
+
+toolchain = {'name': 'intel', 'version': '2015b'}
+toolchainopts = {'pic': True}
```

Comment:
Sure, it actually only applies to static libraries, I think?

It basically means that your static library is built such that it can be linked in other libraries, which in turn are linked dynamically?